shopify_integration/hook_events/connected_app.py

```diff
@@ -0,0 +1,16 @@
+from typing import TYPE_CHECKING
+from urllib.parse import urljoin
+
+import frappe
+
+if TYPE_CHECKING:
+	from frappe.integrations.doctype.connected_app.connected_app import ConnectedApp
+
+
+def validate_redirect_uri(connected_app: "ConnectedApp", method: str):
+	if "shopify" in connected_app.provider_name.lower():
```

Comment:
What if we checked for links to a Shopify Settings document?

Reply:
Most likely, we'll create Connected App first, and then add it as a link to the Shopify Settings document